api/pokemon.tsx

```diff
@@ -1,10 +1,11 @@
-interface Pokemon {
+export interface Pokemon {
   name: string;
-  weight: number;
-  height: number;
-  base_experience: number;
-  sprites: [object];
+  weight: string;
+  height: string;
+  base_experience: string;
+  sprites: [{ front_default: string; front_shiny: string }];
   url: string;
+  types: [{ type: { name: string } }];
```

Comment:
you can also write `ReadonlyArray<{...}>` to enforce that this list cannot be modified (you'd have to copy it first e.g. with `[...types]`)

Sometimes it's also nice to extract types, e.g.:
```ts
interface Type {
  name: string;
}
types: ReadonlyArray<Type>;
```
but it's not needed in such simple scenarios :)